src/Collection/Fulfilment/OrderCollection.php

```diff
@@ -104,8 +104,8 @@ function (Order $order) {
      */
     private function getShipmentOptions(AbstractDeliveryOptionsAdapter $deliveryOptions): array
     {
-        $dateTime     = new DateTime($deliveryOptions->getDate());
-        $deliveryDate = $deliveryOptions->getDate()
+        $dateTime     = $deliveryOptions->getDate() ? new DateTime($deliveryOptions->getDate()) : null;
+        $deliveryDate = $dateTime
```

Comment:
Het wordt een beetje lang maar 2 ternaries achter elkaar is nog raarder, ik zie liever dit:
```php
$deliveryDate = $deliveryOptions->getDate()
    ? $new DateTime($deliveryOptions->getDate())->format(AbstractOrder::DATE_FORMAT_FULL)
    : null;

```

En als je dat niet mooi vindt kan je het nog naar een losse functie refactoren

Reply:
Ik heb het nu voor de stijl maar even in een if gezet, want blijkbaar snapt php het niet als je die format functie gelijk achter new DateTime() plakt